### ISSUE TITLE
node install.js

### ISSUE THREAD

**Issue by m-ghoneimy** [electron-userland/electron-prebuilt#239]
Hi,
I'm trying to install Electron on 64-bit Windows 10 using Windows CMD. When I enter the following command:

`npm install electron-prebuilt --save-dev`

it works fine till it comes to "node install.js" then it hangs for a long time then outputs the following list of errors:

```
D:\CS_Projects\lnl\node_modules\electron-prebuilt\install.js:22
  throw err
  ^

Error: read ECONNRESET
    at exports._errnoException (util.js:1034:11)
    at TLSWrap.onread (net.js:580:26)
npm WARN lnl@1.0.0 No description
npm WARN lnl@1.0.0 No repository field.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! electron-prebuilt@1.4.13 postinstall: `node install.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the electron-prebuilt@1.4.13 postinstall script 'node install.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the electron-prebuilt package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node install.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs electron-prebuilt
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls electron-prebuilt
npm ERR! There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\mghon\AppData\Roaming\npm-cache\_logs\2017-04-10T00_54_08_806Z-debug.log

D:\CS_Projects\lnl>
```

I tried to solve it but I haven't come to any solution. Please help me.
Thanks!

**Comment by MarshallOfSound** [comment]
`ECONNRESET` means that while downloading the Electron binaries your internet connection gave up / the download failed.  Can you try on a different network?

**Comment by m-ghoneimy** [comment]
I solved it! I just downgraded node.js to v6.1.0 and it worked fine! Thanks! :)

**Comment by saifalfalah** [comment]
Same thing happened to me on NodeJS 8.1.3. Resolved it by downgrading to 6.11.0

**Comment by lekeCoder** [comment]
Well for me I would say that if you have time and patience, wait for it to complete work in its time but if not simply follow what @m-ghoneimy and @saifalfalah suggested.
